test/verifyImageLabels.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+
+VCS_REF=$(docker inspect -f '{{ index .Config.Labels "org.label-schema.vcs-ref"}}' wurstmeister/kafka)
+echo "VCS_REF=$VCS_REF"
+if [ -z "$VCS_REF" ] || [ "$VCS_REF" = "unspecified" ]; then
+  echo "org.label-schema.vcs-ref is empty or unspecified"
+  exit 1
+fi
+if ! git cat-file -e "$VCS_REF^{commit}"; then
+  echo "$VCS_REF Not a valid git commit"
+  exit 1
+fi
+
+BUILD_DATE=$(docker inspect -f '{{ index .Config.Labels "org.label-schema.build-date"}}' wurstmeister/kafka)
+echo "BUILD_DATE=$BUILD_DATE"
+if ! date -d "$BUILD_DATE"; then
```

Comment:
This will fail on MacOS because of the BSD version of date. However, I think this is OK for now - but ideally should be run from inside a container like the other tests so that it can be executed from any machine.